lib/rpmsg/rpmsg_virtio.c

```diff
@@ -797,15 +800,17 @@ void rpmsg_deinit_vdev(struct rpmsg_virtio_device *rvdev)
 	struct rpmsg_device *rdev;
 	struct rpmsg_endpoint *ept;
 
-	rdev = &rvdev->rdev;
-	while (!metal_list_is_empty(&rdev->endpoints)) {
-		node = rdev->endpoints.next;
-		ept = metal_container_of(node, struct rpmsg_endpoint, node);
-		rpmsg_destroy_ept(ept);
-	}
+	if (rvdev) {
+		rdev = &rvdev->rdev;
+		while (!metal_list_is_empty(&rdev->endpoints)) {
+			node = rdev->endpoints.next;
+			ept = metal_container_of(node, struct rpmsg_endpoint, node);
+			rpmsg_destroy_ept(ept);
+		}
 
-	rvdev->rvq = 0;
-	rvdev->svq = 0;
+		rvdev->rvq = 0;
```

Comment:
Why the indentation change here?

Reply:
The code is within the if conditional now, so it should be lined up as such.

Reply:
Got it. Looks good to go.